### PR TITLE
Add backward_ros to moveit_common

### PR DESCRIPTION
**Pull request by tylerjw**
### Description

This adds backward_ros to all moveit packages that use the moveit_common cmake function.  This is a really useful tool and building it in by default seems like a really nice feature.

https://github.com/pal-robotics/backward_ros/tree/foxy-devel

### CODE REVIEW

**Comment by tylerjw** [comment]
> This is great! Are we sure that this doesn't have any performance impacts? From what I've read it wouldn't change the binaries at all as it fully relies on the debug info.

This will result in slightly larger binaries (one more library linked in) but other than that I don't believe so.

**Review by vatanaksoytezer — APPROVED** [review]
I'm also in favor of this, this will help a lot!

**Comment by v4hn** [comment]
O.O . This is literally a dream come true :angel:

**Review by v4hn — APPROVED** [review]

File: moveit_common/cmake/moveit_package.cmake [hunk 1]
```diff
@@ -32,6 +32,9 @@ macro(moveit_package)
   find_package(ament_cmake REQUIRED)
   ament_package_xml()
 
+  # Enable backward_ros on every moveit package
+  find_package(backward_ros REQUIRED)
```
Comment (v4hn) [inline moveit_common/cmake/moveit_package.cmake:36]: It doesn't make much difference for a well-setup structure, but QUIET would be enough here I guess?

```suggestion
  find_package(backward_ros QUIET)
```

Comment (henningkayser) [reply moveit_common/cmake/moveit_package.cmake:36]: I agree

**Review by henningkayser — APPROVED** [review]

**Pull request merged**